dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDetailsIvyDevWfBean.java

```diff
@@ -158,6 +159,23 @@ private boolean canResume() {
     return isActivator || isWorker;
   }
 
+  public boolean showInfoBanner() {
+    return selectedTask.getState() == TaskState.RESUMED || selectedTask.getState() == TaskState.DONE;
+  }
+
+  public String getInfoBannerSeverity() {
+    IUser user = ISession.current().getSessionUser();
+    return user.getName().equals(selectedTask.getWorkerUser().getName()) ? "warn" : "info";
+  }
+
+  public String getInfoBannerMessage() {
+    return switch (selectedTask.getState()) {
```

Comment:
Task done involves the following states: TaskState.DONE, TaskState.READY_FOR_JOIN, TaskState.JOINING, TaskState.JOIN_FAILED,
Task in progress involves the following states: TaskState.CREATED, TaskSTate.RESUMED, TaskState.PARKED

Tasks done:
1. Current user has worked on it: "You already have completed the task"
2. Other user: "Task has already been completed by user xyz" 

Tasks in progress:
1. Current user is working on the task: "You are currently working on the task in a different session. You may reset the task and start it again if you have closed your browser."
2. Other user is working on the task: "You cannot work on the task because user xyz is currently working on it."

Task Destroyed (TaskState.DESTROYED):
"You cannot work on the task because it was destroyed"